src/sst/elements/mercury/components/nic.h

```diff
@@ -18,11 +18,12 @@
 #include <mercury/common/component.h>
 
 #include <sst/core/event.h>
-#include <sst/core/eli/elementbuilder.h>
+//#include <sst/core/eli/elementbuilder.h>
 #include <sst/core/interfaces/simpleNetwork.h>
 
+#include <mercury/components/node_fwd.h>
 #include <mercury/components/node_base_fwd.h>
-#include <mercury/components/operating_system_fwd.h>
+#include <mercury/components/operating_system_api.h>
```

Comment:
```suggestion
 #include <mercury/components/node_fwd.h>
 #include <mercury/components/node_base_fwd.h>
-#include <mercury/components/operating_system_api.h>
 #include <mercury/common/thread_safe_new.h>
 #include <mercury/common/node_address.h>
 #include <mercury/common/timestamp.h>
@@ -41,6 +41,8 @@
 namespace SST {
 namespace Hg {
 
+class OperatingSystemAPI;
+
 class NicEvent :
```

Should the nic know about the OperatingSystemAPI?  Either way, it doesn't need the full declaration if all we're doing is making a pointer to it.  This happens a few times across the changes e.g., node, node_base, etc.